README.md

```diff
@@ -28,10 +28,9 @@ With pip (Python>=3.11):
 pip install browser-use
 ```
 
-install playwright:
-
+Install Playwright: 
```

Comment:
This is causing [confusion](https://github.com/browser-use/browser-use/issues/1020#issuecomment-2722695671).